homeassistant/components/stream/core.py

```diff
@@ -118,6 +118,10 @@ def _render_hls_template(self, last_stream_id: int, render_parts: bool) -> str:
         if self.hls_playlist_complete:
             return self.hls_playlist_template[0]
         if not self.hls_playlist_template:
+            # Logically EXT-X-DISCONTINUITY makes sense above the parts, but Apple's
```

Comment:
Are the changes here in core related to the bigger change? They appear like a separate cleanup.